apps/server/src/modules/files-storage/uc/files-storage-delete.uc.spec.ts

```diff
@@ -0,0 +1,314 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { MikroORM } from '@mikro-orm/core';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { HttpService } from '@nestjs/axios';
+import { ForbiddenException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Counted, EntityId } from '@shared/domain';
+import { AntivirusService } from '@shared/infra/antivirus/antivirus.service';
+import { fileRecordFactory, setupEntities } from '@shared/testing';
+import { Logger } from '@src/core/logger';
+import { AuthorizationService } from '@src/modules/authorization';
+import { S3ClientAdapter } from '../client/s3-client.adapter';
+import { FileRecordParams } from '../controller/dto';
+import { FileRecord, FileRecordParent } from '../entity';
+import { PermissionContexts } from '../files-storage.const';
+import { FilesStorageMapper } from '../mapper';
+import { FilesStorageService } from '../service/files-storage.service';
+import { FilesStorageUC } from './files-storage.uc';
+
+const buildFileRecordsWithParams = () => {
+	const userId = new ObjectId().toHexString();
+	const schoolId = new ObjectId().toHexString();
+
+	const fileRecords = [
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text.txt' }),
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text-two.txt' }),
+		fileRecordFactory.buildWithId({ parentId: userId, schoolId, name: 'text-tree.txt' }),
+	];
+
+	const params: FileRecordParams = {
+		schoolId,
+		parentId: userId,
+		parentType: FileRecordParent.User,
+	};
+
+	return { params, fileRecords, userId };
+};
+
+describe('FilesStorageUC delete methods', () => {
+	let module: TestingModule;
+	let filesStorageUC: FilesStorageUC;
+	let filesStorageService: DeepMocked<FilesStorageService>;
+	let authorizationService: DeepMocked<AuthorizationService>;
+	let orm: MikroORM;
+
+	const createRequestParams = (schoolId: EntityId, userId: EntityId) => {
```

Comment:
Is there a reason why you've put this method (and the one below) in the describe scope while buildFileRecordsWithParams is in the file scope?

Reply:
No reason for it. Moved them in the file scope.